http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -117,10 +125,29 @@ final case class CreateStorageContainerRequest(workspaceId: WorkspaceId,
 final case class CreateStorageContainerResult(resourceId: WsmControlledResourceId)
 final case class WorkspaceDescription(id: WorkspaceId,
                                       displayName: String,
+                                      spendProfile: String,
                                       azureContext: Option[AzureCloudContext],
                                       gcpContext: Option[GoogleProject]
 )
 
+//Landing Zone models
+final case class LandingZone(landingZoneId: UUID,
+                             billingProfileId: UUID,
+                             definition: String,
+                             version: String,
+                             createdDate: String
+)
+final case class ListLandingZonesResult(landingZones: List[LandingZone])
+
+final case class LandingZoneResource(resourceId: String,
```

Comment:
I think not all of these are required. It looks like `resourceType` and `region` are always there, but `resourceName`, `resourceParentId`, and `resourceId` should be optional (a resource will either have a parent+name, or an id).

Here's a sample API response:

```
{
  "id": "e2c33d00-4479-43b8-9a61-efbd7cd857d2",
  "resources": [
    {
      "purpose": "SHARED_RESOURCE",
      "deployedResources": [
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.batch/batchaccounts/lzf6326b75a9c58320e58437",
          "resourceType": "microsoft.batch/batchaccounts",
          "region": "eastus2"
        },
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.containerservice/managedclusters/lz454127da142e705afb28ada",
          "resourceType": "microsoft.containerservice/managedclusters",
          "region": "eastus2"
        },
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.dbforpostgresql/servers/lzba3fe011758486ab362256f4998a49cf5f8402b959bd7ea3c299d4a372d95",
          "resourceType": "microsoft.dbforpostgresql/servers",
          "region": "eastus2"
        },
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.relay/namespaces/lzb1637a0994ffa0c41087aaab55e8c71f5c85f03f3446b549",
          "resourceType": "microsoft.relay/namespaces",
          "region": "eastus2"
        },
        {
          "resourceId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/microsoft.storage/storageaccounts/lz494c20d50d031572b4f301",
          "resourceType": "microsoft.storage/storageaccounts",
          "region": "eastus2"
        }
      ]
    },
    {
      "purpose": "AKS_NODE_POOL_SUBNET",
      "deployedResources": [
        {
          "resourceType": "DeployedSubnet",
          "resourceName": "AKS_SUBNET",
          "resourceParentId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/Microsoft.Network/virtualNetworks/lz77e23c6133c90e3e813cb742a66a88f47fa65d811720e1b4d53ab3092f85ca",
          "region": "eastus2"
        }
      ]
    },
    {
      "purpose": "WORKSPACE_COMPUTE_SUBNET",
      "deployedResources": [
        {
          "resourceType": "DeployedSubnet",
          "resourceName": "COMPUTE_SUBNET",
          "resourceParentId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/Microsoft.Network/virtualNetworks/lz77e23c6133c90e3e813cb742a66a88f47fa65d811720e1b4d53ab3092f85ca",
          "region": "eastus2"
        }
      ]
    },
    {
      "purpose": "POSTGRESQL_SUBNET",
      "deployedResources": [
        {
          "resourceType": "DeployedSubnet",
          "resourceName": "POSTGRESQL_SUBNET",
          "resourceParentId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/Microsoft.Network/virtualNetworks/lz77e23c6133c90e3e813cb742a66a88f47fa65d811720e1b4d53ab3092f85ca",
          "region": "eastus2"
        }
      ]
    },
    {
      "purpose": "WORKSPACE_BATCH_SUBNET",
      "deployedResources": [
        {
          "resourceType": "DeployedSubnet",
          "resourceName": "BATCH_SUBNET",
          "resourceParentId": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-rtnov14-20221114121959/providers/Microsoft.Network/virtualNetworks/lz77e23c6133c90e3e813cb742a66a88f47fa65d811720e1b4d53ab3092f85ca",
          "region": "eastus2"
        }
      ]
    }
  ]
}
```